docs/source/technical_tutorials/access/accessing_LVIS_data.ipynb

```diff
@@ -0,0 +1,1765 @@
+{
```

Comment:
<p>We could keep this consistent with other tutorials and do a "./data" directory?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Okay</p>